src/generics/Starter.java

```diff
@@ -0,0 +1,39 @@
+package generics;
+
+
+import java.util.*;
+
+/**
+ * Created by Komatoz on 17.01.2018.
+ */
+public class Starter {
+
+
+    public static void main(String[] args) {
+        System.out.println("Finish task3");
+    }
+
+    public static <T> int countEven(List<? extends Number> list){
+        int result = 0;
+        for (Number num : list) {
+            if (Math.IEEEremainder(Double.valueOf(num.toString()), 2) == 0) result++;
+        }
+        return result;
+    }
+
+    public static <T> void swapListElement(List<T> list1, List<T> list2, int i){
+        list1.set(i, list2.get(i));
+    }
+
+    public static <T> void switchListElement(List<T> list1, List<T> list2, int i ){
+        T temp = list1.get(i);
+        list1.set(i, list2.get(i));
+        list2.set(i, temp);
+    }
+
+    public static <T> T maxElement(List<T> list){
+        TreeSet<T> set = new TreeSet<>(list);
+        return set.last();
```

Comment:
Can you try this code:
`class Person {}`
`Starter.maxElement(Arrays.asList(new Person(), new Person()))`